roles/init_dbserver/tasks/init_dbserver.yml

```diff
@@ -59,9 +59,17 @@
     - not validate_only|bool
   become: true
 
+- name: Set run_initdb to false if it's a PGD server and has use_physical_backup
+  set_fact:
+    run_initdb: false
+  when:
+    - hostvars[inventory_hostname].pgd.use_physical_backup is defined
+    - hostvars[inventory_hostname].pgd.use_physical_backup|bool
+
```

Comment:
I think these checks should be within the `supported_roles.py` file

Reply:
Not sure. I am following you. I will ping you to get more information.

Reply:
As discussed, resolving this.